airflow/models/taskinstance.py

```diff
@@ -1320,9 +1320,9 @@ def set_duration(self):
 
     def xcom_push(
             self,
-            key,
-            value,
-            execution_date=None):
+            key: str,
+            value: Any,
+            execution_date: Optional[datetime] = None) -> None:
```

Comment:
Are these changes required to make this PR pass tests? If not it is preferred to not change files that you don't have to for the subject of the PR.

Reply:
I have removed the type annotation.